src/components/schema/KeyAutoComplete/index.tsx

```diff
@@ -47,9 +47,10 @@ function KeyAutoComplete({ disabled, onChange, value }: Props) {
     const [localCopyValue, setLocalCopyValue] = useState<string[]>([]);
     const [inputValue, setInputValue] = useState('');
 
-    // We only want to all edit during Capture create/edit.
+    // We only want to all edit during capture create/edit and derivation create.
     const entityType = useEntityType();
-    const editKeyAllowed = entityType === 'capture';
+    const editKeyAllowed =
+        entityType === 'capture' || entityType === 'collection';
```

Comment:
Thinking out loud - we should really look into "inverting" this pattern. All over the place we fetch out the type and then do some logic with it. I wonder if we made a context where you fetch what the entity type can do would be better. Have not put a lot of thought into this idea yet.